docs/02-app/02-api-reference/02-file-conventions/instrumentation.mdx

```diff
@@ -45,9 +60,82 @@ export function register() {
 }
 ```
 
+### `onRequestError` (optional)
+
+You can optionally export an `onRequestError` function to track and send **server** errors to an observability tool.
+
+```ts filename="instrumentation.ts" switcher
+import { type Instrumentation } from 'next'
+
+export const onRequestError: Instrumentation.onRequestError = (
```

Comment:
Since TS doesn't support complex types in function declarations, we have to use an arrow functions.